packages/eslint-plugin-gestalt/src/eslintASTHelpers.js

```diff
@@ -0,0 +1,313 @@
+// @flow strict
```

Comment:
This is awesome. My one nit would be to split these out to different files, but that's your call. It would help identify dead code and make these helpers more discoverable. Either way this is _super_ helpful!

Reply:
Splitted `Helpers` from `Fixers`
DONE